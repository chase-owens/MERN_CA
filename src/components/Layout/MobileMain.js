import React from 'react';
import Nav from '../Nav/Nav';

const MobileMain = () => {
  return (
    <div>
      <Nav />
    </div>
  );
};

export default MobileMain;
