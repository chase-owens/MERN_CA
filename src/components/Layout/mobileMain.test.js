import React from 'react';
import MobileMain from './MobileMain';
import { shallow } from 'enzyme';

describe('MobileMain', () => {
  it('renders without crashing', () => {
    const mobilemain = shallow(<MobileMain />);
    expect(mobilemain).toMatchSnapshot();
  });
});
