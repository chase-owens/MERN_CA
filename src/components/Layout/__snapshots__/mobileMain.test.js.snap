// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MobileMain renders without crashing 1`] = `
<div>
  <WithTheme(WithWidth(Connect(Nav))) />
</div>
`;
